packages/adapter-netlify/README.md

```diff
@@ -42,13 +42,19 @@ You may build your app using functionality provided directly by SvelteKit withou
 
 ### Using Netlify Redirect Rules
 
+> [Netlify docs here](https://docs.netlify.com/routing/redirects/)
+
+Netlify supports declarative redirects via a `_redirects` file. Place it in the `static/` folder, NOT in your project root next to `netlify.toml`.
+
 During compilation a required "catch all" redirect rule is automatically appended to your `_redirects` file. (If it doesn't exist yet, it will be created.) That means:
 
 - `[[redirects]]` in `netlify.toml` will never match as `_redirects` has a [higher priority](https://docs.netlify.com/routing/redirects/#rule-processing-order). So always put your rules in the [`_redirects` file](https://docs.netlify.com/routing/redirects/#syntax-for-the-redirects-file).
```

Comment:
just fyi - i think this is inaccurate - the doc actually says "Rules in the _redirects file are always processed first, followed by rules in the Netlify configuration file."

in other words you CAN put stuff in netlify.toml, AND in _redirects, its just that the latter overrides the former in cases of conflict.

Reply:
I believe it's true when you consider the catch-all route that gets added